girder/constants.py

```diff
@@ -205,6 +206,7 @@ class SettingDefault(object):
             'users': []
         },
         SettingKey.CONTACT_EMAIL_ADDRESS: 'kitware@kitware.com',
+        SettingKey.PRIVACY_NOTICE: 'https://www.kitware.com/privacy',
```

Comment:
I'm not sure if we want to point the default to Kitware's privacy policy, but it makes as much sense as pointing the default contact email to kitware@kitware.com, which we currently do. I think this is OK.